DashSync/shared/Models/Wallet/DSWallet.m

```diff
@@ -185,12 +185,6 @@ - (instancetype)initWithUniqueID:(NSString *)uniqueID andAccounts:(NSArray<DSAcc
     if (!(self = [self initWithChain:chain])) return nil;
     self.uniqueIDString = uniqueID;
     __weak typeof(self) weakSelf = self;
-    self.seedRequestBlock = ^void(SeedCompletionBlock seedCompletion) {
-        //this happens when we request the seed
-        NSString *seed = [weakSelf seedPhrase];
-        NSData *seedData = [[DSBIP39Mnemonic sharedInstance] deriveKeyFromPhrase:seed withPassphrase:nil];
-        seedCompletion(seedData, false);
-    };
```

Comment:
This callback was added for CrowdNode a year ago so that we're able to send transactions without prompting the user.
Given that there is no asynchronous work happening inside the block, we can make it a method instead. This will simplify things for CoinJoin a lot.